src/components/ListRow/index.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { spacing, colors } from "../../theme";
```

Comment:
I recommend importing these individually as this will bring in all other unused imports from "../../theme"